src/pages/stepsCalculator/stepsCalculator.tsx

```diff
@@ -0,0 +1,148 @@
+        heightCm = Number((ev.target as HTMLInputElement).value);
+        if (heightCm <= 0) {
+            return 0;
+        }
+    }
+
+    function placeHolder() {
+        // return steps.toString();
+        const updateSteps = document.querySelector('#result');
+        if (updateSteps != null) {
+            if (metric == false && heightFt > 0 && heightIn > 0) {
+                // imperial calculations
+                // height in IN * convert = inches per stride
+                // IN per stride / 12 = feet per stride
+                // 5280 / feet per stride = steps per mile
+                steps = miles * (5280 / ((((heightFt * 12) + heightIn) * strideConverter) / 12));
+            }
+            else if (metric == true) {
+                // metric calculations
+            }
+            updateSteps.innerHTML = Math.round(steps).toString();
+        }
+        // updateSteps.innerHTML = steps.toString();
+        return;
+    }
+
+    return (
+        <>
+            <IonHeader>
+                <IonToolbar>
+                    <IonTitle>
+                        Steps Calculator
+                    </IonTitle>
+                </IonToolbar>
+            </IonHeader>
+
+            <IonContent className="ion-padding">
+                <IonItem>
+                    <IonLabel position="floating">Number of miles walked</IonLabel>
+                    <IonInput type="number" onInput={
+                        (event: any) => {
+                            calculate(event);
+                        }
+                    }></IonInput>
+                    <IonRouterLink slot="helper" href="./manualLoggingSteps">Return to steps logging</IonRouterLink>
+                </IonItem>
+                <IonGrid>
+                    <IonRow>
+                        <IonCol>
+                            <IonItem>
+                                <IonLabel position="floating">Optional: Height ft</IonLabel>
+                                <IonInput type="number" onInput={
+                                    (event: any) => {
+                                        getHeightFt(event);
+                                    }
+                                }></IonInput>
+                            </IonItem>
+                        </IonCol>
+                        <IonCol>
+                            <IonItem>
+                                <IonLabel position="floating">Optional: Height In</IonLabel>
+                                <IonInput type="number" onInput={
+                                    (event: any) => {
+                                        getHeightIn(event);
+                                    }
+                                }></IonInput>
+                            </IonItem>
+                        </IonCol>
+                    </IonRow>
+                </IonGrid>
+                <IonCol>
+                    <IonButton onClick={placeHolder}>Submit</IonButton>
```

Comment:
I love how you've thought ahead here. This will make future tasks super easy.

Reply:
Thanks! Mostly just want to do some basic testing